ikarus/finiteelements/mechanics/materials/muesli/mueslimaterials.hh

```diff
@@ -0,0 +1,133 @@
+// SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+/**
+ * \file materials.hh
+ * \brief Header file for material models in Ikarus finite element mechanics.
+ * \ingroup  mechanics
+ */
+
+#pragma once
+
+#if ENABLE_MUESLI
+
+  #include <ikarus/finiteelements/mechanics/materials/muesli/mueslifinite.hh>
+  #include <ikarus/finiteelements/mechanics/materials/muesli/mueslihelpers.hh>
+  #include <ikarus/finiteelements/mechanics/materials/muesli/mueslismall.hh>
+
+namespace Ikarus::Materials::Muesli {
+
+// Alias for small strain Muesli materials
+using LinearElasticity            = muesli::elasticIsotropicMaterial;
+using LinearAnisotropicElasticity = muesli::elasticAnisotropicMaterial;
+using LinearOrthotropicElasticity = muesli::elasticOrthotropicMaterial;
+
+// Alias for finite strain Muesli materials
+using StVenantKirchhoff = muesli::svkMaterial;
```

Comment:
Do we really need this indirection? I would just use their namespace and we should not introduce our own `Muesli` namespace